src/users/context/users.tsx

```diff
@@ -217,6 +221,8 @@ export const UsersProvider: FC<Props> = React.memo(({children}) => {
         window.dataLayer.push({
           event: GTM_USER_REMOVED,
         })
+
+        if (userId == currentUserId) window.location.href = CLOUD_URL
```

Comment:
I think you'll get dinged by the linter on this one - we require brackets on all control statements. I'm not sure if there's a lint rule for it but if there isn't we strongly prefer no single line control expressions, so instead of

```ts
if (userId == currentUserId) { window.location.href = CLOUD_URL }
```

```ts
if (userId == currentUserId) {
  window.location.href = CLOUD_URL
}
```

Reply:
Thanks much -- I got that changed now!